README.md

```diff
@@ -92,16 +92,16 @@ notably slow and lacks the advanced detection capabilities that come with Komac.
 |------------------------------------------|:------:|:------------:|:--------------------------------------------------------------------:|
 | Parameters                               |   ✅    |      ✅       |                                  ❌                                   |
 | Works without Git                        |   ✅    |      ✅       |                                  ❌                                   |
-| Optimised manifest ordering [^1]         |   ✅    |      ❌       |                                  ✅                                   |
+| Optimised manifest ordering [^1]         |   ✅    |      ✅️       |                                  ✅                                   |
 | Fully cross-platform                     |   ✅    |      ❌       |                               Limited                                |
 | Full MSI value retrieval                 |   ✅    |   Partial    |                               Partial                                |
 | Linux & macOS MSI support                |   ✅    |      ❌       |                                  ❌                                   |
 | Full MSIX value retrieval                |   ✅    |   Partial    |      Partial - https://github.com/Trenly/winget-pkgs/issues/180      |
 | Get information from GitHub              |   ✅    |      ❌       |                                  ❌                                   |
 | Formatted GitHub release notes retrieval |   ✅    |      ❌       |                                  ❌                                   |
 | Release date identification              |   ✅    |      ❌       |                                  ❌                                   |
+| No telemetry                             |   ✅    |      ✅️ [^4]  |                                  ✅                                   |
```

Comment:
I think the distinction of 'by default' is important. A lot of people weren't happy that winget had telemetry by default when compared to existing tools like chocolatey. If you have to physically do something to disable it, then I wouldn't really consider it 'no telemetry'.